test/k8s-integration/driver-config.go

```diff
@@ -127,10 +127,12 @@ func generateDriverConfigFile(testParams *testParameters) (string, error) {
 		snapshotClassName = "no-volumesnapshotclass"
 	}
 
-	caps = append(caps, "pvcDataSource")
+	if strings.Contains(testParams.storageClassFile, "sc-standard") {
+		caps = append(caps, "pvcDataSource")
+	}
 	minimumVolumeSize := "5Gi"
 	numAllowedTopologies := 1
-	if testParams.storageClassFile == regionalPDStorageClass {
+	if testParams.storageClassFile == regionalPDStorageClass && !strings.Contains(testParams.storageClassFile, "sc-extreme") {
```

Comment:
This extreme check is redudent, because the regional storage class filename is already checked.